### PR TITLE
model: view_type is undocumented in 8.0

### PR DESCRIPTION
**Pull request by sbidoul**
and described as obsolete in odoo dev cookbook

@olivier-laurent

### CODE REVIEW

**Comment by adrienpeiffer** [comment]
:+1:

File: bobtemplates/odoo/model/views/+model.name_underscored+.xml.bob [hunk 1]
```diff
@@ -75,7 +75,6 @@
         <field name="name">{{{ model.name_camelwords }}}</field> <!-- TODO -->
         <field name="res_model">{{{ model.name_dotted }}}</field>
         <field name="view_mode">tree,form</field>
-        <field name="view_type">form</field>
```
Comment (Olivier-LAURENT) [inline bobtemplates/odoo/model/views/+model.name_underscored+.xml.bob]: I suggest to keep this field. Ok, it's the defaut value but at least tagging is ready to be modified if any. It should be nice to do the same for `context` and `domain`. No ?

**Comment by Olivier-LAURENT** [comment]
On master this field is still in the `act_window` model

File: bobtemplates/odoo/model/views/+model.name_underscored+.xml.bob (hunk 1 above)
Comment (sbidoul) [inline bobtemplates/odoo/model/views/+model.name_underscored+.xml.bob]: I'll add context and domain in another PR.

**Comment by sbidoul** [comment]
> On master this field is still in the act_window model

Yes, but I could not find it in the documentation and the odoo dev cookbook says *The window action's view_type attribute is mostly obsolete by now. The alternative to the default form is tree, which causes the groups lists to render a hierarchical tree. Don't confuse this attribute with the view_mode attribute used and explained earlier, which actually decides which types of views are used.* 

That makes me think we should leave it out, as it is seldom used.

**Comment by Olivier-LAURENT** [comment]
Thus ok for this field: +1

In addition only the menu must be protected by `{{% if model.view_menu %}}`,  maybe the action would be referenced on a button from another view...

**Pull request merged**